.github/workflows/ci.yml

```diff
@@ -220,21 +220,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # test recent mypy versions on 3.10 and mypy 1.0.1 on all supported python versions
-        # if you change the value in the lockfile add the old value to this list
-        mypy-version: ['1.0.1', 'lockfile', '1.2.0', '1.4.1', '1.5.1']
-        python-version: ['3.10']
+        # test the latest version on all supported Python versions and the rest on 3.12
+        mypy-version: ['1.10.1', '1.11.2']
+        python-version: ['3.12']
         include:
-          - mypy-version: 'lockfile'
+          - mypy-version: '1.12.0'
```

Comment:
Can we add a var like `'latest'` above so that we can just update that in one place instead of all of these?

Reply:
This can't be done easily with GH actions currently, apart from using job outputs and inputs.

Reply:
Np, can leave as is then.